### PR TITLE
Global strip_tags Validation

### PR DESCRIPTION
**Pull request by TFWIsaac**
Hey, 

i have just added a tip for global / project-wide `strip_tags` validation

### CODE REVIEW

File: routing.md [hunk 1]
```diff
@@ -675,3 +675,49 @@ Route::controller(UsersController::class)->group(function () {
...
+protected $middleware = [
+// \App\Http\Middleware\TrustHosts::class,
+\App\Http\Middleware\TrustProxies::class,
+\Illuminate\Http\Middleware\HandleCors::class,
+\App\Http\Middleware\PreventRequestsDuringMaintenance::class,
+\Illuminate\Foundation\Http\Middleware\ValidatePostSize::class,
+\App\Http\Middleware\TrimStrings::class,
+\Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull::class,
+// This:
+\App\Http\Middleware\StripTags::class,
```
Comment (KeithBush) [inline routing.md:717]: If you're going to do this, then your middleware should appear before ConvertEmptyStringsToNull.
Passing null to strip_tags is deprecated and triggers a warning.

**Review by KeithBush — COMMENTED** [review]
It is probably worth nothing the potential downsides to using something like this tip.  strip_tags will remove anything that **looks** like a tag, not just actual tags. Depending on your data, it may remove potentially large chunks of input.  IE: 

```php
> strip_tags('Therefore we can conclude 5<100 blah blah')
= "Therefore we can conclude 5"
```

In my opinion, changing this to use htmlspecialcharts instead of strip_tags would provide more reliable results for your users.  Input they entered with tags would be displayed back the way they entered it but without triggering any sort of XSS

**Comment by tvbeek** [comment]
To be honest I think this is not a good tip. 
Strip tags is for displaying content.

It can create a fake sense of security while it isn't.
Another downside is that it can alter your input on ways that aren't suspected of or looks like a tag. 

The positive part of this tip is showing the way to update all request input fields. That is a nice trick 👍

**Comment by TFWIsaac** [comment]
Okay my bad, this was not my best idea, So i close this PR

**Pull request closed without merging**